fedimintd/src/bin/distributedgen.rs

```diff
@@ -55,10 +55,6 @@ enum Command {
         /// Our node name, must be unique among peers
         #[arg(long = "name")]
         name: String,
-
-        /// The password that encrypts the configs, will prompt if not passed in
-        #[arg(long = "password")]
```

Comment:
`#[arg(env = "ENV_NAME")]` is what you want. Must enabled `env` `feature` on `clap` too.

Reply:
I actually have the same thing pending in my docker branch. I named the env var `FEDIMINT_PASSWORD`.

Reply:
Is this better than calling `std::env::var("FM_PASSWORD")` in a single place?